config/queue.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Default Queue Connection Name
+    |--------------------------------------------------------------------------
+    |
+    | Laravel's queue API supports an assortment of back-ends via a single
+    | API, giving you convenient access to each back-end using the same
+    | syntax for every one. Here you may define a default connection.
+    |
+    */
+
+    'default' => env('QUEUE_CONNECTION', 'sync'),
+
+    /*
+    |--------------------------------------------------------------------------
+    | Queue Connections
+    |--------------------------------------------------------------------------
+    |
+    | Here you may configure the connection information for each server that
+    | is used by your application. A default configuration has been added
+    | for each back-end shipped with Laravel. You are free to add more.
+    |
+    | Drivers: "sync", "database", "beanstalkd", "sqs", "redis", "null"
+    |
+    */
+
+    'connections' => [
+        'redis' => [
+            'driver' => 'redis',
+            'connection' => 'default',
+            'queue' => env('REDIS_QUEUE', 'default'),
+            'retry_after' => 90,
+            'block_for' => null,
+            'after_commit' => false,
+        ],
+        'redis_high' => [
+            'driver' => 'redis',
+            'connection' => 'default',
+            'queue' => 'high',
+            'retry_after' => 90,
+            'block_for' => null,
+            'after_commit' => false,
+        ],
+        'redis_low' => [
+            'driver' => 'redis',
+            'connection' => 'default',
+            'queue' => 'low',
+            'retry_after' => 90,
+            'block_for' => null,
+            'after_commit' => false,
+        ],
```

Comment:
Although we're defining these new connections, no job is using them. The jobs have different queues associated, but since we're not declaring different connections, they're all being sent to the default connection (`redis`).

Reply:
I have defined the low or high in some jobs: https://github.com/FogosPT/fogosapi/blob/feature/14-priority-queue/app/Observers/IncidentObserver.php#L23

Reply:
I've noticed it, but the queue name is not the connection name, which was what we are defining here. We can have one connection that handles multiple queues. So, to put it simply: we should keep one Redis connection.